src/linux/py_proc.h

```diff
@@ -68,6 +68,21 @@ union {
 
 
 // ----------------------------------------------------------------------------
+static Elf64_Addr
+_py_proc_get_base64(Elf64_Ehdr * ehdr, void * elf_map)
+{
+  Elf64_Addr res = (Elf64_Addr) 0 - 1;
```

Comment:
Is `0 - 1` really required (as opposed to just `-1`)?

Reply:
Probably better and more explicit to use `UINT64_MAX` and `UINT32_MAX`.

Reply:
I think that `(Elf64/32_Addr) -1` is fine. Perhaps one could `#define INVALID_BASE ((Elf64/32_Addr) -1)` and use that to check for a valid base address.

Reply:
Yes, `(Elf64_Addr) -1` and `(Elf32_Addr) -1` would work.

I'm not so sure about using a `#define` however (it would probably be necessary to define or for 32 bit and one for 64 bit).

Is there any downside of using `UINT64_MAX`? I find this quite clear.

Reply:
> Is there any downside of using `UINT64_MAX`? I find this quite clear.

No, that's fine too. Mine was just a suggestion.